quizquestions.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
UI improvement: you can remove "read instructions carefully" in quiz questions.
![Screenshot 2022-02-20 at 2 31 01 PM](https://user-images.githubusercontent.com/79623546/154835641-f305ecb7-9c70-4dcd-82e4-3245b3f23139.png)

Reply:
But why to remove it😅